apptools/naming/pyfs_context.py

```diff
@@ -232,7 +231,7 @@ def _lookup(self, name):
                     # If the load fails then we create a generic file resource
                     # (the idea being that it might be useful to have access to
                     # the file to see what went wrong).
-                    except:
+                    except Exception:
```

Comment:
I would personally prefer leaving a `# noqa: EXXX` here instead of adding a bare `Exception`. That way, someone searching for flake8 ignores in the futures will notice this and maybe find time to investigate - instead of assuming that this is the behavior originally intended.